### ISSUE TITLE
New lists suggestions

### ISSUE THREAD

**Issue by Marek33** [notracking/hosts-blocklists#375]
I'm not sure, if these lists can be used, since first two are probably dead, but, you could check the other three since they look quite interesting: **Anti-WebMiner, MinerBlock list, Malwaredomains.com, malwaredomainlist.com** and maybe the **Shallalist**. However, I have no idea, how is it, with the copyright, since this is just a suggestion, since I do have a lot of lists, which are not updated for a long time, but they are not used in your pack, where they probably should be.

Also, the blacklist.site has some great lists, from which I would definitely use the ransomware​​​​​ one, and probably other ones, but there seems to be quite a lot of false positives. The others are either not that important or too big like the malware list.

**Comment by notracking** [comment]
Shalla - License and number of false positives makes it unsuited
Anti-webminer - Not maintained
MinerBlock list - Don't know where to find it, nocoin list is already included
Malwaredomains.com - already included
malwaredomainlist.com - already included

Thanks for the suggestions, but having direct links to potential new lists helps a lot. You can check [https://github.com/notracking/hosts-blocklists/blob/master/SOURCES.md](SOURCES.md) for lists that have already been included. Searching the closed issues will also help.

**Comment by Marek33** [comment]
@notracking  I understand, though I didn't find the list of used lists, on the Wiki, so I wasn't sure, if I should mention them or not. However, when I disabled those two lists, and seen that the number of filters has been lowered, then I added them, even if those lists aren't that great, as they used to be.

Anti-WebMiner and MinerBlock seems to be good lists, so that's why I would like to see them added, and here is the MinerBlock: https://github.com/xd4rker/MinerBlock Its also not much active, but both lists, could have some sites, which should be blocked.

That link gives me this message: **This page isn’t working**
